backend/src/main/java/gov/cdc/usds/simplereport/api/model/TestEventExport.java

```diff
@@ -304,32 +288,32 @@ public String getOrderingFacilityCounty() {
 
 	@JsonProperty("Ordering_facility_name")
 	public String getOrderingFacilityName() {
-		return org.getFacilityName();
+		return facility.getFacilityName();
```

Comment:
To maintain our current format (`"${organization.name}, ${facility.name}"`) this needs to include the organization name

Reply:
I am not sure on this I think the the Facility name the is associated with the CLIA number might include what looks like the organization name 🤔

Reply:
Confirmed with @aliciabeckett-gov this should remain as return `facility.getFacilityName();` but our current export needs to be updated